src/plugins/saved_objects_management/public/management_section/objects_table/components/import_mode_control.tsx

```diff
@@ -120,7 +120,7 @@ export const ImportModeControl = ({
       options={[overwriteEnabled, overwriteDisabled]}
       idSelected={overwrite ? overwriteEnabled.id : overwriteDisabled.id}
       onChange={(id: string) => onChange({ overwrite: id === overwriteEnabled.id })}
-      disabled={createNewCopies}
+      disabled={createNewCopies && !isLegacyFile}
```

Comment:
This fixes a bug that I didn't run into until I changed `createNewCopies` to be enabled by default on the client side.